tests/acceptance/TestHelpers/GraphHelper.php

```diff
@@ -43,6 +43,14 @@ public static function isUUIDv4(string $id): bool {
 		return (bool)preg_match($regex, $id);
 	}
 
+	/**
+	 *
+	 * @return string
+	 */
+	public static function getFileUUIDv4Regex(): string {
+		return self::getUUIDv4Regex() . '%24' . self::getUUIDv4Regex() . '%21' . self::getUUIDv4Regex();
+	}
+
```

Comment:
```suggestion
```

I think this won't be necessary we can assert uuidv4 regex with encoded value in the json schema something as:
```
fileId=%uuidv4_pattern%%24%uuidv4_pattern%%21%uuidv4_pattern%
```